src/CLI/clitree/cli-xml/sonic_types.xml

```diff
@@ -41,6 +41,13 @@ limitations under the License.
         help=""
         />
     <!--=======================================================-->
+    <PTYPE
+        name="LAG_ID"
+        method="integer"
+        pattern="1..128"
```

Comment:
Can you delete the LAG_ID existing as well? Looks like there is a LAG_ID with pattern 0..9999 already existing.